app/controllers/recipes_controller.rb

```diff
@@ -38,27 +51,33 @@ def destroy
     @recipe.destroy
 
     respond_to do |format|
-      format.html { redirect_to recipes_url }
+      format.html { redirect_to recipes_url, notice: 'Recipe successfully removed' }
       format.json { head :no_content }
     end
   end
```

Comment:
Deleting the recipe function doesn't work at the moment. Besides, as an authenticated user, I shouldn't be able to see the remove button for other recipes and shouldn't be able to delete them. Please fix this part.

Reply:
Got it. Resolving it right away